meta-balena-common/recipes-containers/balena/balena_git.bb

```diff
@@ -103,14 +103,13 @@ do_install() {
 	ln -sf balena-engine ${D}/${bindir}/balena
 	ln -sf balena-engine ${D}/${bindir}/balenad
 	ln -sf balena-engine ${D}/${bindir}/balena-containerd
-	ln -sf balena-engine ${D}/${bindir}/balena-containerd-shim
+	ln -sf balena-engine ${D}/${bindir}/balena-containerd-shim-runc-v2
 	ln -sf balena-engine ${D}/${bindir}/balena-containerd-ctr
 	ln -sf balena-engine ${D}/${bindir}/balena-runc
 	ln -sf balena-engine ${D}/${bindir}/balena-proxy
 
 	ln -sf balena-engine ${D}/${bindir}/balena-engine-daemon
 	ln -sf balena-engine ${D}/${bindir}/balena-engine-containerd
-	ln -sf balena-engine ${D}/${bindir}/balena-engine-containerd-shim
```

Comment:
are there any other symlinks here that shouldn't be required?

Reply:
I never understood why (or if) we need those ambiguous symlinks (like `balena-engine-daemon` and `balenad`), but have been keeping them because Chesterton’s Fence.

Reply:
That's what I thought, so I guess we have to hope it's okay to remove one that doesn't exist anymore.

Reply:
Right... but in the case of the shim I am not *that* much worried in the (unlikely?) case of something breaking, because the old shim is deprecated anyway, so this hypothetical thing was going to break anyway soon. But yeah, no 100% safe.

Reply:
close enough for me